Nickel/Interfaces/IHasCustomCardTraits.cs

```diff
@@ -4,5 +4,5 @@ namespace Nickel;
 
 public interface IHasCustomCardTraits
 {
-	public IReadOnlySet<string> GetInnateTraits(State state);
+	public IReadOnlySet<ICardTraitEntry> GetInnateTraits(State state);
```

Comment:
Just to note: I originally intentionally kept this as a Set of `string`s instead of a set of `ICardTraitEntry` so you could very easily add a trait to a card by just returning a set with `SomeOtherMod::TheTrait` out of an implementation of this method rather than having to first fetch the trait entry via `ModHelper` &c (since that is really annoying to do in a `Card` subclass unless you save it somewhere `static`).